repository-data/webfiles/src/main/resources/site/freemarker/common/project-update-feed.ftl

```diff
@@ -0,0 +1,188 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+<#include "macro/stickyNavSections.ftl">
+<#include "macro/stickyNavYears.ftl">
+<#include "macro/stickyNavTags.ftl">
+<#include "macro/hubBox.ftl">
+<#include "macro/documentHeader.ftl">
+<#include "macro/stickyGroupBlockHeader.ftl">
+<#include "macro/stickyNavYears.ftl">
+
+<@hst.setBundle basename="rb.generic.headers,rb.generic.labels,rb.doctype.projectupdate "/>
+
+<#-- Add meta tags -->
+<#include "macro/metaTags.ftl">
+<@metaTags></@metaTags>
+
+<#assign monthNames = monthsOfTheYear()?reverse />
+
+<#--Group the documents by month  -->
+<#assign eventGroupHash = {} />
+<#list pageable.items as item>
+    <#if item.publiclyAccessible && item.updateTimestamp?size gt 0>
+        <@fmt.formatDate value=item.updateTimestamp.time type="Date" pattern="MMMM" timeZone="${getTimeZone()}" var="key" />
+        <#assign eventGroupHash = eventGroupHash + {  key : (eventGroupHash[key]![]) + [ item ] } />
+    </#if>
+</#list>
+
+<#function getOrganisationLinks>
+    <#assign links = [] />
+
+<#--    <#list pageable.items as key>-->
+<#--        <#if key.publiclyAccessible>-->
+<#--            <#assign links += [{ "key" : slugify(key.organisation.displayName), "title": key.organisation.displayName }] />-->
+<#--        </#if>-->
+<#--    </#list>-->
+
+    <#return links />
+</#function>
+
+<#function getTypeOfupdateLinks>
+    <#assign links = [] />
+
+<#--    <#list pageable.items?sort_by("typeOfUpdate") as key>-->
```

Comment:
This bit looks strange. Is it commented out?

Reply:
it is commented out. It was waiting for some BE work but it has since been agreed to go ahead without, so I will remove this.